holo-vrrp/src/instance.rs

```diff
@@ -403,6 +412,16 @@ impl Instance {
             let _ = net.net_tx_packetp.send(msg);
         }
     }
+
+    /// an instance is an owner if the ip address of the parent interface is
+    /// part of the virutal addresses held by the instance.
+    pub(crate) fn check_is_owner(&self, interface_sys: &InterfaceSys) -> bool {
+        self.config
+            .virtual_addresses
+            .iter()
+            .any(|addr| interface_sys.addresses.contains(addr));
+        false
```

Comment:
You should return the result of the last expression instead of always returning false